### DISCUSSION TITLE
Github Copilot Activation Error

### DISCUSSION THREAD

**Discussion by ghost** [community/community#73476] [GitHub Education] [answered]
### Hi everyone,

Hey guys,

Regarding the Github Student Developer Pack, I have activated my account and have also verified my student status, but I am not able to activate my Github Copilot free subscription that is provided with the pack.

**Comment by karkir0003** [comment]
@Akash1134, do you have any thoughts here?

**Comment by GigaSmurf** [comment]
Click on your profile on the top right, and go to where it says GitHub Copilot. You can activate it there.

**Reply by ghost** [reply]
I tried that, but it just says this:

<img width="1257" alt="Screenshot 2023-11-05 at 2 46 34 PM" src="https://github.com/community/community/assets/121770960/f8c5c641-90f4-40a4-b2ae-97dfa0a2d86e">

**Reply by GigaSmurf** [reply]
When did you verify your status. There might be some delay. Can you see your other benefits?

**Reply by ghost** [reply]
About a month ago lol, how can I check the other benefits?

**Reply by GigaSmurf** [reply]
Go to GitHub Education and, in the top navigation bar, click Benefits. Under "Individuals", click Get student benefits. https://education.github.com/

**Reply by ghost** [reply]
Yep, checked that and it just gives me this, and it just goes into a loop when i click any of the options. When I tried to reverify my status, it would just say that I am already verified and I can't do it again

<img width="1151" alt="Screenshot 2023-11-05 at 3 26 08 PM" src="https://github.com/community/community/assets/121770960/2033a6de-5196-4ff3-abb3-a04f8185d349">

**Reply by GigaSmurf** [reply]
are you able to see the benefits through this link or by going to offers: https://education.github.com/pack/offers
<img width="1340" alt="image" src="https://github.com/community/community/assets/45702710/c7733413-01df-4dbc-8c35-53655137243a">
<img width="1358" alt="image" src="https://github.com/community/community/assets/45702710/2ce2a04b-78ec-40c6-b768-819fa08cacc1">

**Reply by ghost** [reply]
Nope, it just brings me to the page of the image I sent above.

**Reply by GigaSmurf** [reply]
Also when i click on benefits, it looks like this:
<img width="1389" alt="image" src="https://github.com/community/community/assets/45702710/d78a3c78-83aa-4bcd-99e6-4c659bf44125">

**Reply by GigaSmurf** [reply]
I guess try submitting your information again like it says. See if it does anything.

**Reply by ghost** [reply]
I can't even resubmit my information, it says that I am already approved

**Reply by GigaSmurf** [reply]
What does your status say on this link: https://education.github.com/discount_requests/pack_application

you should be able to resubmit here maybe.

**Reply by ghost** [reply]
Yea, it says I am verified, but when resubmitting my info here it doesn't let me

**Reply by GigaSmurf** [reply]
Does it say approved, like this on the right side?
<img width="355" alt="image" src="https://github.com/community/community/assets/45702710/d0a5fa51-1b87-46b7-80b4-64d14225d589">

**Reply by ghost** [reply]
Yea, it says that exactly which is why I am confused

**Reply by GigaSmurf** [reply, marked as answer]
Wow, that is strange. Maybe try contacting GitHub support. I remember that there was like a delay for me as well and I remember trying some random stuff.

But you can also check some of these discussions:
https://github.com/orgs/community/discussions/58268
https://github.com/orgs/community/discussions/56029
https://github.com/orgs/community/discussions/55426
https://github.com/orgs/community/discussions/57998

**Reply by ghost** [reply]
Ah ok, thanks so much for your help!

**Reply by ghost** [reply]
Yea sure!
